src/components/Sweep/SweepExecute.tsx

```diff
@@ -0,0 +1,196 @@
+
+  const handleFromChange = useCallback(
+    (e) => {
+      setFromAddress(e.target.value);
+    },
+    [setFromAddress]
+  );
+
+  useEffect(() => {
+    const validAFromAddress = ethers.utils.isAddress(fromAddress);
+    const getBalance = async () => {
+      const balanceEth = await contractBalanceOf(fromAddress);
+      setMaxAmount(balanceEth[1]);
+    };
+
+    const validToAddress = ethers.utils.isAddress(to);
+    const getAllowance = async () => {
+      const balanceWei = await contractAllowance(fromAddress, to);
+      setAllowance(balanceWei);
+    };
+
+    if (validAFromAddress) {
+      try {
+        getBalance();
+      } catch (e) {
+        console.log(e);
+      }
+    }
+    if (validAFromAddress && validToAddress) {
+      try {
+        getAllowance();
+        setValid(true);
+      } catch (e) {
+        console.log(e);
+      }
+    } else {
+      setValid(false);
+    }
+  }, [fromAddress, to]);
+
+  const handleToChange = useCallback(
+    (e) => {
+      setTo(e.target.value);
+    },
+    [setTo]
+  );
+
+  const handleAmountChange = useCallback(
+    (e) => {
+      setAmount(e.target.value);
+    },
+    [setAmount]
+  );
+
+  const handleTransfer = useCallback(async () => {
+    const validAddress = ethers.utils.isAddress(fromAddress);
+    if (validAddress) {
+      try {
+        setLoading(true);
+        await changeNetwork(42161);
+
+        await contractTransferFrom(fromAddress, to, amount);
+
+        notification.success({
+          message: `From ${fromAddress} to ${to} ${amount} Magic has been transferred succesffuly. `,
+        });
+      } catch (e) {
+        console.error(e);
+        const error = (e as any)?.message || 'Error';
+        notification.error({ message: error });
+      } finally {
+        setLoading(false);
+      }
+    }
+  }, [fromAddress, to, amount, changeNetwork]);
+
+  const setAmountToMax = () => setAmount(maxAmount);
+
+  return (
+    <Container>
+      <Typography.Title level={5} className="subtitle">
+        Transfer Magic ✨ Tokens
+      </Typography.Title>
+      <Form.Item
+        name="fromAddress"
+        label="From Address:"
+        rules={[
+          { required: true, message: 'From Ethereum Address is required' },
+          { validator: (_, value) => ethereumAddressValidator(value) },
+        ]}
+        className="title"
+      >
+        <Input
+          type="text"
+          placeholder="The Ethereum Address Magic Token's will be transfered from"
+          value={fromAddress}
+          onChange={handleFromChange}
+          required={true}
+        />
+      </Form.Item>
+      <Form.Item name="Balance" label="Balance">
+        <Input type="number" placeholder="The Maximum Amount avalible tokens" value={maxAmount} disabled={true} />{' '}
+      </Form.Item>
+      <Form.Item
+        name="toAddress"
+        label="To Address:"
+        rules={[
+          { required: true, message: 'To Ethereum Address is required' },
```

Comment:
No need to be so verbose in the error message. It will be displayed right under the input anyway so it's enough to say `Address is required` or `This field is required`